### DISCUSSION TITLE
Use server sent data a temporary keys

### DISCUSSION THREAD

**Discussion by pke** [i18next/i18next#2136] [Q&A]
I have the scenario where a server response contains translation keys I want to use to use pluralisation with on the client.
How can I bind a `t` function to a temporary set of keys, that are not loaded from local resources?
Or is the pluralisation function somehow exposable so I can use it with a set of keys?

**Comment by jamuhl** [comment]
no...you will have to add those keys to the resource store...even just if temporary...https://www.i18next.com/overview/api#resource-handling

best practice would be to hold those translations in i18next anyway and let the server just pass down needed keys instead of fulltext

**Reply by pke** [reply]
That's not possible to hold the keys in the client. The server sends all texts down in each response.
I'll find a pluralisation library then and not use i18next for that.

**Comment by jamuhl** [comment]
you can call https://github.com/i18next/i18next/blob/master/src/PluralResolver.js#L153 on `i18next.services.pluralResolver`

eg.: `i18next.services.pluralResolver.getSuffix('en-US', 5); // -> returns _other`

**Reply by pke** [reply]
The full key resolution would then be up to me I guess? So for any given count there is always only one possible suffix and if a key with this suffix does not exist, then I am looking for a key without any suffix?

**Reply by jamuhl** [reply]
right
